pkg/certificate/authorization/authorizations_test.go

```diff
@@ -622,7 +633,11 @@ func TestNewClient(t *testing.T) {
 	})
 }
 
-func newTestAuthDB(ctx *testcontext.Context) (*DB, error) {
+func newTestAuthDB(t *testing.T, ctx *testcontext.Context) *DB {
 	dbURL := "bolt://" + ctx.File("authorizations.db")
-	return NewDB(dbURL, false)
+	db, err := NewDB(dbURL, false)
+	if err != nil {
```

Comment:
to have the same result just use `require.NoError`

Reply:
It seems to be a convention that test helper code uses `t.Fatal` inside of a condition rather than using `require`.

Reply:
`require` would be completely fine. I think some of the testing code was not using `require`, because we didn't know about it. Initially everything was using `assert`, so to stop the test you needed to explicitly call `Fatal`.

Reply:
:+1: